components/login/index.jsx

```diff
@@ -45,12 +45,21 @@ const useStyles = makeStyles((theme) => createStyles(styles(theme)));
 
 export default function LoginForm() {
   const bem = useBem(useStyles());
+  // FIXME: remove after pod migration is completed
+  const [shouldDisplayWarning, setShouldDisplayWarning] = useState(false);
 
   const { persist } = useReturnUrl();
   const { login: sessionLogin } = useSession();
 
   const login = useCallback(
     (oidcIssuer) => {
+      if (
+        oidcIssuer.includes("login.inrupt.com") ||
+        oidcIssuer.includes("broker.pod.inrupt.com")
+      ) {
+        setShouldDisplayWarning(true);
+        return;
+      }
```

Comment:
```suggestion
      } else {
         setShouldDisplayWarning(false);
      }
```

Reply:
If `shouldDisplayWarning` is set to false by default, and only changed to true if this condition is true, do we need this?

Reply:
Well, you can type in one idp, then get the warning, then type in a different one

Reply:
but if that happens you're redirected, as this is the login function and it is not called on value change, so the warning will stay there even when typing and won't clear until it is submitted